.github/workflows/build-and-publish-wheel.yml

```diff
@@ -33,6 +33,13 @@ jobs:
           export POETRY_HTTP_BASIC_PYPI_PASSWORD="${{ secrets.pypi_token }}"
           make publish
 
+      - name: Build and publish packages to private PYPI
```

Comment:
Does it build the package twice for public and private publishings?

Reply:
Yes - for simplicity. I guess we can: 

a) add another Make target of publish private-and-public
b) make it dependent on both the `publish` and `private-publish`

I guess Make should make a diamond graph then and build it only once. 

Do you think this is a big deal?

Reply:
I think it's not a big deal

Reply:
Ok